salt/modules/gpg.py

```diff
@@ -493,7 +504,12 @@ def create_key(
 
 
 def delete_key(
-    keyid=None, fingerprint=None, delete_secret=False, user=None, gnupghome=None
+    keyid=None,
+    fingerprint=None,
+    delete_secret=False,
+    user=None,
+    gnupghome=None,
+    use_passphrase=False,
```

Comment:
Prior to this change the we would have used the passphrase by default. I think this should still default to true so we don't break existing behavior on people.

Reply:
Thank you for the reminder. The original behaviour was restored and the tests updated to reflect this.